docs/DeveloperGuide.adoc

```diff
@@ -1850,6 +1898,34 @@ Use case ends.
 +
 Use case resumes at step 3.
 
+=== Statistic Report Window
+
+[discrete]
+==== Use case: UC23 - Displaying a statistic report
+
+*MSS*
+
+1. User requests to generate and display a statistic report.
+2. TA-Tracker retreives the list of completed sessions, and students.
+3. TA-Tracker processes the data retreived.
+4. TA-Tracker opens a Statistic window and displays the processed data to the user.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. The user did not specify a specific module code.
```

Comment:
And then this extension should be "the user specified a module code"
1a1. TA-Tracker only retrieve data from the sessions and students associated with the module
Use case ends

or whichever way you want

Reply:
Ah good point

Reply:
Would I then still need the subcase for if the module code is not valid?